### ISSUE TITLE
Missing GetNetState on PocketBook Basic 624, segfault

### ISSUE THREAD

**Issue by 000exploit** [koreader/koreader#6548]
* KOReader version: v2020.08.1
* Device: PocketBook Basic 624

#### Issue
App can't start, `libinkview.so` doesn't have `GetNetState` symbol

#### Steps to reproduce
1. Unarchive and copy `applications` to USB root
2. Launch koreader

##### `crash.log`
```
./luajit: frontend/device/pocketbook/device.lua:202: /usr/lib/libinkview.so: undefined symbol: GetNetState
stack traceback:
	[C]: in function '__index'
	frontend/device/pocketbook/device.lua:202: in function 'isWifiOn'
	frontend/ui/network/manager.lua:90: in function 'init'
	frontend/ui/network/manager.lua:657: in main chunk
	[C]: in function 'require'
	frontend/apps/reader/modules/readerdictionary.lua:14: in main chunk
	[C]: in function 'require'
	frontend/apps/reader/readerui.lua:31: in main chunk
	[C]: in function 'require'
	./reader.lua:274: in main chunk
	[C]: at 0x00013f31
Segmentation fault
```

**Comment by Frenzie** [comment]
Duplicate of https://github.com/koreader/koreader/issues/6546

**Comment by 000exploit** [comment]
Fixed in #6559, closing issue
